docs/developer-guide/syntax-reference.rst

```diff
@@ -647,6 +666,52 @@ Records written into the stream are not timestamp-ordered with respect to other
 Therefore, the topic partitions of the output stream may contain out-of-order records even
 if the source stream for the query is ordered by timestamp.
 
+
+INSERT VALUES
+-----------
+
+**Synopsis**
+
+.. code:: sql
+
+    INSERT INTO <stream_name|table_name> [(column_name [, ...]])]
+      VALUES (value [,...]);
+
+**Description**
+
+Produce a row into an existing stream or table and its underlying topic based on
+explicitly specified values. The first ``column_name`` of every schema is ``ROWKEY``, which
+defines the corresponding kafka key - if the source specifies a ``key`` and that column is present
+in the column names for this insert statement, the values are expected to match, otherwise the
+value will be duplicated into the value (or conversely from the value into the ``ROWKEY``).
```

Comment:
"the ROWKEY value will be duplicated"?

Reply:
I meant that the contents of the ROWKEY will be copied into the value for the key column. Any suggestions in phrasing?

Reply:
Maybe: 
>  If the source specifies a ``key`` and that column is present in the column names for this INSERT statement then the values are expected to match, otherwise the value from ``ROWKEY`` will be copied into the value of the key column (or conversely from the key column into the ``ROWKEY`` column).